README.md

```diff
@@ -88,6 +90,19 @@ let topConstraint = (imageView.topAnchor == container.topAnchor)
 let edgeConstraints = (button.edgeAnchors == container.edgeAnchors).all
 ```
 
+## Creating Inactive Constraints
+
+By default, Anchorage returns active layout constraints. If you'd rather return inactive constraints for use with the [`NSLayoutConstraint.activate(_:)` method](https://developer.apple.com/reference/uikit/nslayoutconstraint/1526955-activate) for performance reasons, you can do it like this:
+
+```swift
+let constraints = Anchorage.inactiveConstraints {
```

Comment:
I think a version of this that returns active constraints would also be helpful, both to activate a group at once for performance reasons and to just collect a set of constraints to manage as a group.